src/pages/EnablePayments/ActivateStep.js

```diff
@@ -21,24 +27,64 @@ const defaultProps = {
     userWallet: {},
 };
 
-const ActivateStep = props => (
-    <ScreenWrapper>
-        <HeaderWithCloseButton
-            title={props.translate('activateStep.headerTitle')}
-            onCloseButtonPress={() => Navigation.dismissModal()}
-            shouldShowBackButton
-            onBackButtonPress={() => Navigation.goBack()}
-        />
-        <View style={[styles.mh5, styles.flex1]}>
-            {props.userWallet.tierName === CONST.WALLET.TIER_NAME.GOLD && (
-                <Text>{props.translate('activateStep.activated')}</Text>
-            )}
-            {props.userWallet.tierName === CONST.WALLET.TIER_NAME.SILVER && (
-                <Text>{props.translate('activateStep.checkBackLater')}</Text>
-            )}
-        </View>
-    </ScreenWrapper>
-);
+class ActivateStep extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.renderGoldWalletActivationStep = this.renderGoldWalletActivationStep.bind(this);
+    }
+
+    renderGoldWalletActivationStep() {
+        return (
+            <>
+                <View style={[styles.pageWrapper, styles.flex1, styles.flexColumn, styles.alignItemsCenter, styles.justifyContentCenter]}>
+                    <Icon
+                        src={Illustrations.TadaBlue}
+                        height={100}
+                        width={100}
+                        fill={defaultTheme.iconSuccessFill}
+                    />
+                    <View style={[styles.ph5]}>
+                        <Text style={[styles.mt5, styles.h1, styles.textAlignCenter]}>
+                            {this.props.translate('activateStep.activatedTitle')}
+                        </Text>
+                        <Text style={[styles.mt3, styles.textAlignCenter]}>
+                            {this.props.translate('activateStep.activatedMessage')}
+                        </Text>
+                    </View>
+                </View>
+                <FixedFooter>
+                    <Button
+                        text={this.props.translate('common.continue')}
+                        onPress={PaymentMethods.continueSetup}
+                        style={[styles.mt4]}
+                        iconStyles={[styles.mr5]}
+                        success
+                    />
+                </FixedFooter>
+            </>
+        );
+    }
+
+    render() {
+        return (
+            <ScreenWrapper>
+                <HeaderWithCloseButton
+                    title={this.props.translate('activateStep.headerTitle')}
+                    onCloseButtonPress={() => Navigation.dismissModal()}
+                    shouldShowBackButton
+                    onBackButtonPress={() => Navigation.goBack()}
+                />
+                <View style={[styles.mh5, styles.flex1]}>
```

Comment:
```suggestion
                <View style={styles.flex1}>
```

Let's remove the horizontal margin, it seems a little excessive in comparison to the mockup:


| Current      | Mockup | After removing mh5 |
| ----------- | ----------- |----------- |
|   ![Screen Shot 2022-08-22 at 6 59 50 PM](https://user-images.githubusercontent.com/12268372/185953440-0887d197-a3d0-44f6-9159-e015bcf444b9.png)   |   <img src="https://user-images.githubusercontent.com/12268372/178017051-572df5e7-eb0d-4d91-9f85-8fdbf81d01f4.png">    |  ![Screen Shot 2022-08-22 at 6 55 20 PM](https://user-images.githubusercontent.com/12268372/185953247-ad2ccf40-d995-492c-933e-b77e6e1d7a24.png) |